docs/sites.yml

```diff
@@ -2344,4 +2344,12 @@
     - Video 
     - Portfolio 
     - Freelance
-  featured: false
\ No newline at end of file
+  featured: false
+  - title: Mike's Remote List
```

Comment:
Hey @mfosker! 👋Could you unindent (remove the two spaces at the beginning of) that line (`  - title:…`) please? 🙏